### ISSUE TITLE
Cont Issue 33, Scenarios not saving properly

### ISSUE THREAD

**Issue by joellenb** [nathan-rice/ctools#38]
I loaded a saved scenario, went into the Perform Analysis tab, updated the scenario back to what it was prior to the incorrect save (reported earlier), and saved the update. I then went back to load the saved scenario. Its properties are still listed incorrectly in the list of scenarios.
![c-line_performanalysis_01132015](https://cloud.githubusercontent.com/assets/10145737/5727128/be39dbd6-9b2e-11e4-8480-8de5d2a760d4.PNG)
![c-line_loadprevscenario_01132015](https://cloud.githubusercontent.com/assets/10145737/5727129/be40612c-9b2e-11e4-8cdf-d46f8dc1175f.PNG)
As a further test, I exited FireFox, started again, went to the testing site, and loaded the saved scenario. Please see following picture showing that scenario was not saved properly.
![c-line_reloadloadsavedscenario_01132015](https://cloud.githubusercontent.com/assets/10145737/5727204/421a3202-9b2f-11e4-8460-b3d4110ba559.PNG)
And the wrong values show up again in the Perform Analysis box. Although the pollutant is not listed in the saved scenario screen, the following screen shows that the pollutant also is being saved incorrectly.
![c-line_performanalysis_afterreload_01132015](https://cloud.githubusercontent.com/assets/10145737/5727265/affd993a-9b2f-11e4-8084-fcae0be319bc.PNG)

**Comment by michellegrace** [comment]
Originally I thought this was fixed, but after reading through this multiple times it seems like this is still an issue. Maybe I can add another example....

i opened a new browser for C-LINE and left all the default values, i.e. stable & winter.
![screen shot 2015-01-14 at 2 09 39 pm](https://cloud.githubusercontent.com/assets/10077162/5745472/0fa2e6e8-9bf7-11e4-81bc-1ad04a479758.png)

I clicked "save" and it was saved like this....
![screen shot 2015-01-14 at 2 10 39 pm](https://cloud.githubusercontent.com/assets/10077162/5745482/2615df16-9bf7-11e4-81e7-8856fb1062ee.png)

The met condition is wrong (it is listed as Neutral)... so i tried changing the met condition to Convective in the perform analysis window 
![screen shot 2015-01-14 at 2 11 38 pm](https://cloud.githubusercontent.com/assets/10077162/5745497/46a3b44c-9bf7-11e4-8fe8-d76e200c4c26.png)

Then i hit save. This time the met condition was saved correctly...
![screen shot 2015-01-14 at 2 12 01 pm](https://cloud.githubusercontent.com/assets/10077162/5745511/60fb840a-9bf7-11e4-86bc-c230b88c34ab.png)

I tried this multiple times and the same thing happened, in the first "fresh" instance of C-LINE the default met condition is "Stable" and it is always saved as Neutral. If I changed the met condition to "Convective" then back to "Stable" it saves as "Stable".

Tested on treehug 1/14/15 ~2:10pm.
 ...
